sensor_control/navigator_battery_monitor/nodes/navigator_battery_monitor.py

```diff
@@ -90,20 +89,30 @@ def publish_voltage(self, event):
         Publishes the average voltage across all four thrustersto the battery_voltage node as a standard Float32 message and runs
         the voltage_check
         '''
-        if (self.voltage != None):
-            if (len(self.supply_voltages) > 0):
-                self.voltage = sum(self.supply_voltages) / len(self.supply_voltages)
 
-            self.pub_voltage.publish(self.voltage)
-            self.voltage_check()
+        if (len(self.supply_voltages) > 0):
+            self.voltage = sum(self.supply_voltages) / len(self.supply_voltages)
+
+        self.pub_voltage.publish(self.voltage)
+        self.voltage_check()
 
     def voltage_check(self):
         '''
         Checks the battery voltage and raises alarms when it's level drops below the set thresholds. This is intended to protect
         and extend the life of the batteries.
         '''
+
+        # An unknown battery status warning to inform users that thruster feedback is not being published
+        if (self.voltage is None):
```

Comment:
The dashboard will count any message containing "None" towards the timeout counter for the battery voltage. Once this times out (after approximately 15s of None messages), the GUI will display "Unknown" as the battery voltage and change the color of it's display frame from green to red.